apps/spruce/src/components/Banners/TaskSchedulingWarningBanner.tsx

```diff
@@ -0,0 +1,20 @@
+import Banner from "@leafygreen-ui/banner";
+import { taskSchedulingLimitsDocumentationUrl } from "../../constants/externalResources";
+import { StyledLink } from "../styles";
+
+interface TaskSchedulingWarningBannerProps {
+  totalTasks: number;
+}
+const largeNumFinalizedTasksThreshold = 1000;
+
+export const TaskSchedulingWarningBanner: React.FC<
+  TaskSchedulingWarningBannerProps
+> = ({ totalTasks }) =>
+  totalTasks >= largeNumFinalizedTasksThreshold ? (
+    <Banner data-cy="disabled-webhook-banner" variant="warning">
```

Comment:
I think this `data-cy` attribute is wrong and probably unnecessary. We typically use these attributes for our e2e tests so that we can easily reference components but I don't see any e2e tests for this so you can omit it.